tests/test.py

```diff
@@ -87,6 +89,15 @@ def setUp(self):
         cfn_mock.return_value = self.cfn_conn_mock
         boto.cloudformation.connect_to_region = cfn_mock
 
+    def test_get_stack_instance_public_ips(self):
+        cfn = cloudformation.Cloudformation(self.env.aws_profile, 'aws_region')
+        cloudformation.Cloudformation.get_stack_instance_ids = mock.Mock()
+        ec = ec2.EC2(self.env.aws_profile, 'aws_region')
+        with patch.object(ec, 'get_instance_public_ips', return_value=['1.1.1.1', '2.2.2.2']):
+            x = ec.get_stack_instance_public_ips('mock-stack-name')
+            cfn.get_stack_instance_ids.assert_called_with('mock-stack-name')
+            compare(x, ['1.1.1.1', '2.2.2.2'])
```

Comment:
The mocking here looks suspect but I'm not quite sure why... it kind of looks like we aren't actually testing anything here